### DISCUSSION TITLE
Missing BG Alert Not Making Sound on Samsung Galaxy 9 Android 10

### DISCUSSION THREAD

**Discussion by johnwd152202** [NightscoutFoundation/xDrip#2181] [General]
Missing BG Alert Not Making Sound on Samsung Galaxy 9 Android 10

Blood Sugar Alerts work as usual.

**Comment by Navid200** [comment]
Would you please have a look at the following and let me know if it helps?
https://navid200.github.io/xDrip/docs/Silent-alert.html
